cs/Platforms/Client.Wpf/Client.Wpf.csproj

```diff
@@ -6,6 +6,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Opc.UaFx.Client" Version="2.30.0" />
+    <PackageReference Include="Opc.UaFx.Client" Version="2.41.1" />
```

Comment:
We should probably also use multitargeting and add `net6.0-windows` to the target frameworks in this project, since it's already prepared for .NET Core Windows Desktop apps (with `<UseWPF>true</UseWPF>`).

When adding  `net6.0-windows` to `<TargetFrameworks>`, we need to change the Sdk value from `Microsoft.NET.Sdk.WindowsDesktop` to `Microsoft.NET.Sdk`.